src/y0/algorithm/estimation.py

```diff
@@ -110,12 +110,29 @@ def _markov_blanket_overlap(graph: NxMixedGraph, u: Variable, v: Variable) -> bo
 
 
 def is_a_fixable(graph: NxMixedGraph, treatments: Union[Variable, List[Variable]]) -> bool:
+    """Check if the treatments are a-fixable.
```

Comment:
@pnavada sara and I already wrote tests for all of the methods. you can run them with `tox -e py`

Reply:
Great! Thank you @cthoyt 
Will check them out